L1Trigger/ME0Trigger/interface/ME0TriggerBuilder.h

```diff
@@ -51,4 +54,33 @@ class ME0TriggerBuilder {
   std::unique_ptr<ME0Motherboard> tmb_[MAX_ENDCAPS][MAX_CHAMBERS];
 };
 
+template <class T>
+void ME0TriggerBuilder::build(const T* me0Pads, ME0TriggerDigiCollection& oc_trig) {
+  for (int endc = 0; endc < 2; endc++) {
```

Comment:
use the enum values MAX_ENDCAPS and MAX_CHAMBERS here instead of magic numbers

Reply:
@dildick although @kpedro88 has signed this PR I share his comment, I will merge it but please update it in a next round